jose-jwt/Security/Cryptography/EccKey.cs

```diff
@@ -128,7 +128,7 @@ public static CngKey New(byte[] x, byte[] y, byte[] d=null, CngKeyUsages usage=C
 
         public static EccKey Generate(CngKey recieverPubKey)
```

Comment:
you probably would like to rename param name of the method as well.

Reply:
Indeed.
The pull request was updated.